lib/vtls/mbedtls.c

```diff
@@ -805,7 +810,7 @@ mbed_connect_step1(struct Curl_cfilter *cf, struct Curl_easy *data)
     return CURLE_SSL_CONNECT_ERROR;
   }
 
-#ifdef MBEDTLS_SSL_TLS1_3_SIGNAL_NEW_SESSION_TICKETS_ENABLED
+#ifdef HAS_SESSION_TICKETS
```

Comment:
If I understand things correctly, the `HAS_SESSION_TICKETS` might get set with >= 3.6.0 but `MBEDTLS_SSL_TLS1_3_SIGNAL_NEW_SESSION_TICKETS_ENABLED` requires >= 3.6.1 making them not quite interchangeable like this. What do you say @icing, I think you added this ifdef?

Reply:
Right, as I read the mbedTLS changelog, this was added in 3.6.1.